webapp/channels/src/utils/utils.tsx

```diff
@@ -1333,15 +1333,15 @@ export function isValidPassword(password: string, passwordConfig: ReturnType<typ
 }
 
 function isChannelOrPermalink(link: string) {
-    let match = (/\/([^/]+)\/channels\/(\S+)/).exec(link);
+    let match = (/\/([a-z0-9\-_]+)\/channels\/([a-z0-9\-__]+)/).exec(link);
```

Comment:
Were you going to revert these changes? I think these could also include a period as well

Reply:
I have another PR that removes this completely. But you are right, the second part should also allow dots, since it could be a channel name. Will look into it.